From 9c8f651ded7f9d26f4017a5af4eb70bc918892cf Mon Sep 17 00:00:00 2001
From: Aravinda VK <avishwan@redhat.com>
Date: Fri, 30 Oct 2015 16:09:29 +0530
Subject: [PATCH 146/162] geo-rep: Allow setting config remote_gsyncd

Restrictive ssh is not used in containerized environment
where networking configuration is "net=host". SSH Pem keys
pushed to the slave without gsyncd path in it. (Patch #12459)

Actual remote_gsyncd path need to be set to actual path of gsyncd.
With this patch, remote_gsyncd is removed from reserved option list.

Change-Id: Ia2063e4654e378b62b2414bdad21143c86ad1b9a
Signed-off-by: Aravinda VK <avishwan@redhat.com>
BUG: 1275971
Reviewed-on: http://review.gluster.org/12472
Tested-by: NetBSD Build System <jenkins@build.gluster.org>
Tested-by: Gluster Build System <jenkins@build.gluster.com>
Reviewed-by: Saravanakumar Arumugam <sarumuga@redhat.com>
Reviewed-by: Venky Shankar <vshankar@redhat.com>
(cherry picked from commit 7de355b42dc1f8313db3ffc775a0e1708ba85243)
Reviewed-on: http://review.gluster.org/12644
Reviewed-by: Vijay Bellur <vbellur@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/62770
---
 xlators/mgmt/glusterd/src/glusterd-geo-rep.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/xlators/mgmt/glusterd/src/glusterd-geo-rep.c b/xlators/mgmt/glusterd/src/glusterd-geo-rep.c
index 2f2ad86..8006a4d 100644
--- a/xlators/mgmt/glusterd/src/glusterd-geo-rep.c
+++ b/xlators/mgmt/glusterd/src/glusterd-geo-rep.c
@@ -74,7 +74,6 @@ struct gsync_config_opt_vals_ gsync_confopt_vals[] = {
 static char *gsync_reserved_opts[] = {
         "gluster-command-dir",
         "pid-file",
-        "remote-gsyncd"
         "state-file",
         "session-owner",
         "state-socket-unencoded",
-- 
1.7.1

